openmc/material.py

```diff
@@ -914,6 +981,52 @@ def to_xml_element(self, cross_sections=None):
 
         return element
 
+    @classmethod
+    def volume_mix(materials,fracs,name=None):
+        """Mix multiple materials together based on volume fractions
+
+        Parameters
+        ----------
+        materials : Iterable of openmc.Material
+            Materials to combine by volume
+        fracs : Iterable of float
+            Fractions of each material to be combined 
+
+        """
+        fracs = np.asarray(fracs)
+        if np.sum(fracs) != 1:
```

Comment:
You might want to just check if it is sufficiently close to 1. Floating point equality is difficult to ensure.